src/web3/constants.ts

```diff
@@ -17,6 +17,28 @@ export const NETWORK_NAMES = {
   1337: 'Local network',
 };
 
+interface Network {
+  [key: number]: object;
+}
+
+const ADDRESSES_BY_NETWORK: Network = Object.freeze({
+  1: {
+    MTA: '0xa3bed4e1c75d00fa6f4e5e6922db7261b5e9acd2',
+    SaveWrapper: 'TODO',
+  },
+  3: {
+    MTA: '0x273bc479e5c21caa15aa8538decbf310981d14c0',
+    SaveWrapper: '0xF078E3146160459fF7bb09D1D5e2D40238A7bCbe',
+  },
+  42: {
+    MTA: '0xcda64b5d3ca85800ab9f7409686985b59f2b9598',
+    SaveWrapper: 'TODO',
+  },
+});
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const ADDRESSES = ADDRESSES_BY_NETWORK[CHAIN_ID];
```

Comment:
This must be exported, and it should also be used in the codebase, e.g. for getting MTA.